src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  font-weight: 500;
+  font-size: 12px;
+
+  &__list {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+
+    &-item {
+      margin-left: 20px;
+    }
+
+    &-item:first-child {
+      margin-left: 0;
+    }
+
+    &-item:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: #060b35;
+    position: relative;
+    display: inline-block;
+    line-height: 60px;
+
+    &.is-active {
+      color: #00acdc;
+
+      &::after {
+        content: '';
+        display: block;
+        height: 4px;
+        width: 100%;
+        background-color: #00acdc;
```

Comment:
Colors should be defined in variables for reusability and maintainability. Consider moving `#00acdc` to a variable in `variables.scss`.